src/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/StarterWeb-CSharp/Areas/Identity/Data/Migrations/00000000000000_CreateIdentitySchema.cs

```diff
@@ -39,6 +39,12 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 constraints: table =>
                 {
                     table.PrimaryKey("PK_AspNetUserTokens", x => new { x.UserId, x.LoginProvider, x.Name });
+                    table.ForeignKey(
+                        name: "FK_AspNetUserTokens_AspNetUsers_UserId",
+                        column: x => x.UserId,
+                        principalTable: "AspNetUsers",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
```

Comment:
Could you summarise why we need something new here? What new functionality are you adding?

Reply:
The old migrations were obsolete. We need to figure out a better way of generating migrations for the templates. I already spoke with Arthur about this